### PR TITLE
[MM-37755] Idiomatic naming (SMTP, TLS, TCP, XML, CSS, HTML, HTTP)

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
This is a the first PR that changes identifies to use more idiomatic naming.

In the future we might be able to enforce this via golangci-lint.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-37755
https://mattermost.atlassian.net/browse/MM-37761
https://mattermost.atlassian.net/browse/MM-37762
https://mattermost.atlassian.net/browse/MM-37763
https://mattermost.atlassian.net/browse/MM-37767
https://mattermost.atlassian.net/browse/MM-37764
https://mattermost.atlassian.net/browse/MM-37765

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: app/download.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ func (a *App) DownloadFromURL(downloadURL string) ([]byte, error) {
 }
 
 func (s *Server) downloadFromURL(downloadURL string) ([]byte, error) {
-	if !model.IsValidHttpUrl(downloadURL) {
+	if !model.IsValidHTTPUrl(downloadURL) {
```
Comment (streamer45) [inline app/download.go:30]: Should this become `IsValidHTTPURL`?

Comment (hanzei) [reply app/download.go:30]: I'm incident to say no as https://mattermost.atlassian.net/browse/MM-37772 covers that.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

File: app/download.go (hunk 1 above)
Comment (streamer45) [reply app/download.go:30]: That works as well :+1:

**Review by streamer45 — APPROVED** [review]
Looks good :+1:

**Pull request merged**